### ISSUE TITLE
Run periodics daily

### ISSUE THREAD

**Issue by kargakis** [openshift/openshift-azure#864]
The following jobs are not yet fully functional and have been extended to run weekly until the underlying issues are fixed. We should be updating them to run daily selectively as issues are addressed.

- [x] [`periodic-ci-azure-etcdbackuprecovery-e2e`](https://openshift-gce-devel.appspot.com/builds/origin-ci-test/logs/periodic-ci-azure-etcdbackuprecovery-e2e/) (waiting on https://github.com/openshift/openshift-azure/pull/825)
- [x] [`periodic-ci-azure-key-rotation-e2e`](https://openshift-gce-devel.appspot.com/builds/origin-ci-test/logs/periodic-ci-azure-key-rotation-e2e) (waiting on https://github.com/openshift/openshift-azure/pull/794)
- [ ] [`periodic-ci-azure-no-running-under-test`](https://openshift-gce-devel.appspot.com/builds/origin-ci-test/logs/periodic-ci-azure-no-running-under-test) (blocked by https://github.com/openshift/openshift-azure/issues/852)
- [x] [`periodic-ci-azure-scaleupdown-e2e`](https://openshift-gce-devel.appspot.com/builds/origin-ci-test/logs/periodic-ci-azure-scaleupdown-e2e) (waiting on https://github.com/openshift/openshift-azure/pull/795)
- [x] [`periodic-ci-azure-prod-e2e`](https://openshift-gce-devel.appspot.com/builds/origin-ci-test/logs/periodic-ci-azure-prod-e2e) (blocked by the prod OSA RP moving to 3.11)
- [x] [`periodic-ci-openshift-openshift-master-azure-e2e-bushslicer`](https://openshift-gce-devel.appspot.com/builds/origin-ci-test/logs/periodic-ci-openshift-openshift-master-azure-e2e-bushslicer) (waiting on https://github.com/openshift/verification-tests/pull/37)

/area tests

**Comment by kargakis** [comment]
@asalkeld @charlesakalugwu @jim-minter @kwoodson @cjryan

**Comment by kargakis** [comment]
Backup+restore test is green+merged - switching back to daily in https://github.com/openshift/release/pull/2330

**Comment by kargakis** [comment]
Scaleupdown, prod, and key rotation back to daily in https://github.com/openshift/release/pull/2362

**Comment by kargakis** [comment]
Bushslicer job back to daily in https://github.com/openshift/release/pull/2478

**Comment by kargakis** [comment]
Will open a separate issue for the running_under_test=false test.
